tests/tulip.ts

```diff
@@ -0,0 +1,22 @@
+import "mocha";
+
+import { expect } from "chai";
+
+import { MarketAggregator } from "../src";
+import { STAKED_TULIP_MINT } from "../src/sources";
+
+describe("Tulip Staking", () => {
+  it("calculates price of sTULIP vs TULIP", async () => {
+    const aggregator = new MarketAggregator({
```

Comment:
I think this test should be isolated to testing just `StakedTulipMarketSource`. Take a look how xSTEP etc are tested.

Reply:
Fixed test!